Formula/miller.rb

```diff
@@ -29,6 +29,12 @@ def install
     system "make", "install"
   end
 
+  def caveats
+    <<~EOS
+      🃏 Some people call me Maurice 🃏
```

Comment:
Nice, ❤️

Reply:
Thanks!

Are you able to approve this PR, or do I need to do something more first?

Reply:
Yeah, sure, can you squash the commits into one with message as `miller: add caveat message`, I think it is good to 🚀 ! Thanks!

Reply:
I'm used to squashing commits during the PR merge step, but it seems like you're asking me to do it before then.  What's a good way to do that?  Would it require opening a new PR?

Reply:
See https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request#following-up, specifically the section that begins "To make changes based on feedback".